docs/installation.rst

```diff
@@ -97,17 +99,12 @@ which should print out the solutions for the included test images.
 
 You can run the automated test suite with this command (from the repository root dir)::
 
+    # run all tests
     pytest
 
     # skip slow tests
     pytest -m "not slow"
 
-Using cedar-solve as a module in your repository
-
-A specific branch named `no_big_files` is available for practical inclusion as a git submodule
-in your own repository. This does not include the default database and example images, making
-it less than 1 MB. For a specific application you probably want a custom database anyway.
```

Comment:
this branch isn't valid in the cedar-solve fork and we do include the default database in the python wheel. the download size is only 8.1MB for the wheel.